Content.Server/Salvage/SalvageSystem.Runner.cs

```diff
@@ -164,7 +166,16 @@ private void UpdateRunner()
             else if (comp.Stage < ExpeditionStage.MusicCountdown && remaining < TimeSpan.FromMinutes(2))
             {
                 // TODO: Some way to play audio attached to a map for players.
-                comp.Stream = _audio.PlayGlobal(comp.Sound, Filter.BroadcastMap(Comp<MapComponent>(uid).MapId), true);
+                // Parkstation-ExpeditionMusic Start
+                // comp.Stream = _audio.PlayGlobal(comp.Sound, Filter.BroadcastMap(Comp<MapComponent>(uid).MapId), true);
+                var sound = comp.Sound;
+                if (sound == null && _prototypeManager.TryIndex<SoundCollectionPrototype>("ExpeditionCountdownDefault", out var collection))
+                {
+                    sound = collection;
+                    comp.Stream = _audio.PlayGlobal(new SoundPathSpecifier(_random.Pick(sound!.PickFiles), AudioParams.Default.WithVolume(-6)),
+                        Filter.BroadcastMap(Comp<MapComponent>(uid).MapId), true);
+                }
```

Comment:
This looks like the sound will *only* ever play if the Component contains null, at which point it will use the fall back. If the Component *doesn't* contain null, it'll simply skip the part where it actually plays the audio.